addons/account/models/account_invoice.py

```diff
@@ -604,13 +620,77 @@ def action_invoice_sent(self):
             'context': ctx,
         }
 
+    def _invoice_email_split(self, msg):
+        return email_split((msg.get('from') or '') + ',' + (msg.get('cc') or ''))
+
     @api.multi
     @api.returns('self', lambda value: value.id)
     def message_post(self, **kwargs):
         if self.env.context.get('mark_invoice_as_sent'):
             self.filtered(lambda inv: not inv.sent).write({'sent': True})
         return super(AccountInvoice, self.with_context(mail_post_autofollow=True)).message_post(**kwargs)
 
+    @api.model
+    def message_new(self, msg_dict, custom_values=None):
```

Comment:
what is this method doing? please add docstring

Reply:
ok